local-helm/buyingcatalogue/templates/configmap.yaml

```diff
@@ -1,8 +1,8 @@
 {{- $dapiUrl:= ternary (printf "%s.%s" (include "call-nested" (list . "dapi" "dapi.fullname")) .Release.Namespace) .Values.dapi.disabledUrl .Values.dapi.enabled -}}
 {{- $bapiUrl:= ternary (printf "%s.%s" (include "call-nested" (list . "bapi" "bapi.fullname")) .Release.Namespace) .Values.bapi.disabledUrl .Values.bapi.enabled -}}
-{{- $isapiUrl:= ternary .Values.dockerInternalLocalhost .Values.isapi.disabledUrl .Values.isapi.enabled -}}
-{{- $areAllFEComponentsEnabled:= and .Values.mp.enabled .Values.pb.enabled .Values.admin.enabled -}}
-{{- $isapiIssuerUrl:= ternary .Values.dockerInternalLocalhost .Values.isapi.feDisabledIssuerUrl $areAllFEComponentsEnabled -}}
```

Comment:
Would you please answer the question from previous review?

Reply:
as above - good spot